### PR TITLE
New 'newEvent' page, and a form component.

### PR DESCRIPTION
**Pull request by dmorrisSL**
A new URL route to add a new session '/events/new'.
A container component 'NewEvent' that just passes dummy values for now, and has stubbed event handlers.
A new 'display' component NewEventForm.

### CODE REVIEW

File: app/containers/NewEvent.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    submit() {
+        alert('Action on form submit TBD');
+    }
+
+    render() {
+        return (
+          <NewEventForm
+            name={this.props.name}
+            nameValidation={this.props.nameValidation}
```
Comment (pallenSL) [inline app/containers/NewEvent.jsx]: Is this a validation message?  Could we call it somethingMessage or would that be inconsistent with something else?

File: app/components/Events/NewEventForm.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    sendValue(handler) {
+        return (ev) => {
+            handler(ev.target.value);
+        };
+    }
+
+    submitButton() {
+        const buttonAttributes = {};
+
+        const validationMsg =
```
Comment (pallenSL) [inline app/components/Events/NewEventForm.jsx]: Is this (semantically) a bool?  If so could we call it isSomethingInvalid(?)

File: app/components/Events/NewEventForm.jsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                          </td>
+                          <td>{this.props.nameValidation}</td>
+                        </tr>
+                        <tr>
+                          <td><label htmlFor="date">Event date <span className="error">*</span></label></td>
+                          <td>
+                            <input
+                              type="text"
+                              id="date"
+                              name="dob"
```
Comment (pallenSL) [inline app/components/Events/NewEventForm.jsx]: Is that short for "Date of Birth"?

File: app/containers/NewEvent.jsx (hunk 1 above)
Comment (dmorrisSL) [inline app/containers/NewEvent.jsx]: I agree. I will rename it (and others). xxxValidationMessage is much clearer.

File: app/components/Events/NewEventForm.jsx (hunk 2 above)
Comment (dmorrisSL) [inline app/components/Events/NewEventForm.jsx]: Agreed. Commit on its way.

**Comment by pallenSL** [comment]
:sake:

**Pull request merged**